bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/component.tsx

```diff
@@ -110,6 +110,10 @@ const ChatMesssage: React.FC<ChatMessageProps> = ({
     || lastSenderPreviousPage) === message?.user?.userId;
   const isSystemSender = message.messageType === ChatMessageType.BREAKOUT_ROOM;
   const dateTime = new Date(message?.createdAt);
+
+  const msgTime = dateTime.toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true });
```

Comment:
if we're localizing the time string, it should be localized to the same locale the user is currently using, the same way it is done now
en: 2:23 PM
pt-br: 14:20

![2024-01-26_14-23](https://github.com/bigbluebutton/bigbluebutton/assets/3728706/95929d1a-ae86-4ed7-af98-f0ad9d6455be)
![2024-01-26_14-20](https://github.com/bigbluebutton/bigbluebutton/assets/3728706/22434a32-935c-4230-9b5c-3970c018c8da)